hardware-wallet-tests/cardano-app-not-started.ts

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable jest/no-standalone-expect */
+import expect from 'expect';
+
+import {
+  createAndRegisterHardwareWalletChannels,
+  createHardwareWalletConnectionChannel,
+  initLedgerChannel,
+  createSequentialPromptMessages,
+  createGetPublicKeyChannel,
+  ipcRenderer,
+  pollCardarnoApp,
+} from './utils';
+
+export const run = () => {
+  expect.assertions(3);
+
+  createSequentialPromptMessages([
+    'Start test runner',
+    'Plug Ledger Nano S to your computer',
+    'Start Cardano APP on Nano S',
+    'Nano S will prompt to export the public key',
+    'Export the public key',
+  ]);
+
+  createAndRegisterHardwareWalletChannels();
+
+  const publicKeyChannel = createGetPublicKeyChannel();
+  const hardwareWalletConnectionChannel = createHardwareWalletConnectionChannel();
+
+  return new Promise((resolve) => {
+    hardwareWalletConnectionChannel.onReceive(
+      async (params: { path: string }) => {
+        expect(params).toEqual({
+          disconnected: expect.any(Boolean),
+          deviceType: expect.any(String),
+          deviceId: null,
+          deviceModel: expect.any(String),
+          deviceName: expect.any(String),
+          path: expect.any(String),
+        });
```

Comment:
This won't work because at this phase the test script is setting up the channels event listeners so when `initLedgerChannel` channel is invoked the message can be consumed.

Reply:
Can be solved the same way as https://github.com/input-output-hk/daedalus/pull/2900#discussion_r836276875